git-config/Cargo.toml

```diff
@@ -25,7 +25,7 @@ quick-error = "2.0.0"
 [dev-dependencies]
 serial_test = "0.5.1"
 serde_derive = "1.0"
-criterion = "0.3"
+#criterion = "0.3.5"
```

Comment:
Was this disabled intentionally? I think it's used by the benchmark in the `benches` directory.

Reply:
Will fix itself when rebasing against master.